db/models.py

```diff
@@ -50,3 +60,64 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(default=timezone.now)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL,
+                             on_delete=models.CASCADE)
```

Comment:
Ensure that `settings.AUTH_USER_MODEL` is correctly set in your Django settings to point to the custom `User` model if you intend to use it. This is crucial for maintaining the integrity of foreign key relationships.